Contour.appdata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.contour.Terminal</id>
+  <project_license>Apache-2</project_license>
+  <name>Contour Terminal Emulator</name>
+  <summary>Contour is a modern and actually fast, modal, virtual terminal emulator, for everyday use. It is aiming for power users with a modern feature mindset.</summary>
+  <url type="homepage">https://github.com/contour-terminal/contour/</url>
```

Comment:
this is completely insuficient.

missing:
- releases
- screenshot
- content rating
- description